woocommerce/woocommerce-gutenberg-products-block
# Revert "Record a tracks event when merchant toggles cart shipping calculator:"

Reverts woocommerce/woocommerce-gutenberg-products-block#1975

Reverting #1975 as we're not planning on using editor tracks events right now; we'll add customised settings to the WC Tracker snapshot instead - see https://github.com/woocommerce/woocommerce/pull/26084 for details.

Since we aren't firing any editor tracks events, I think it's cleaner to remove the `recordEditorEvent` facility and the test event. 

In the future we may well need tracks events in the block editor – we can implement what's needed then.

#### How to test this PR
- Add cart block to a page or edit a page with cart block.
- Open dev tools network tab.
- Toggle `Shipping calculator` option.
- Should not see tracks event `cart_settings_shipping_calculator_toggle` sent. Can confirm by filtering by pixel.wp.com. (See previous PR for more test details.)

Also test that the cart & checkout blocks work correctly in editor and front end, i.e. no unexpected regressions.
